llvm/test/tools/llvm-profdata/memprof-basic_v3.test

```diff
@@ -0,0 +1,101 @@
+REQUIRES: x86_64-linux
+
+This is a copy of memprof-basict.test with slight changes to check that we can still read v3 of memprofraw.
+
+To update the inputs used below run Inputs/update_memprof_inputs.sh /path/to/updated/clang
```

Comment:
I think this comment is wrong - we can't update the v3 version, is that correct? Nor would we want to.

Reply:
yes, good point! 

done.